### DISCUSSION TITLE
Laravel dusk : (assertRegExp() is deprecated and will be removed in PHPUnit 10.)

### DISCUSSION THREAD

**Discussion by laszlopusztai** [laravel/framework#45592] [Q&A]
Hello,

There is simply dusk test:

```
$browser->assertRouteIs('cars.show', $this->car)
            ->waitForText(...)
            ->assertSee(...);
```

After I shifted Laravel from 6x to 7x, when I run this test, give this warning message:

"assertRegExp() is deprecated and will be removed in PHPUnit 10. Refactor your code to use assertMatchesRegularExpression() instead."

If I remove the assertRouteIs assertion, the test run, without warning.

What should I do? I can not found the solution. Where should I change assertRegExp()?

Laravel 7 
PHPUnit 9.5.27
"laravel/dusk": "^5.6"

Thanks,

**Comment by henzeb** [comment]
The error is in the code base of dusk. So that's where it needs to be fixed. I suggest report a bug at that repo, or maybe even create a PR.

In the meantime, no immediate treat for your product, as phpunit 10 isn't released yet, and laravel 9 is working with 9. And you are way below it.
